storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/dataaccess/RocksDbDao.java

```diff
@@ -62,6 +65,12 @@
 
   Stream<ColumnEntry<Checkpoint, BeaconState>> streamCheckpointStates();
 
+  Stream<ColumnEntry<UnsignedLong, Bytes32>> streamDepositHashes();
+
+  Stream<ColumnEntry<UnsignedLong, DepositData>> streamEth1DepositData();
+
+  Stream<ColumnEntry<UnsignedLong, Eth1BlockData>> streamEth1BlockData();
```

Comment:
Can we annotate these with @MustBeClosed please? Then error prone will enforce that the stream is closed.